programs/eosio-launcher/main.cpp

```diff
@@ -1557,7 +1558,7 @@ launcher_def::launch (eosd_def &instance, string &gts) {
         install_path = specific_nodeos_installation_paths[node_num] + "/";
      }
   }
-  string eosdcmd = install_path + "programs/nodeos/nodeos ";
+  string eosdcmd = install_path + "programs/nodeos/" + string(node_executable_name);
```

Comment:
you lost a trailing space here, and it might be why tests are failing

Reply:
Thank you, I applied it.